content/articles/is-knowledge-graph-just-another-means-of-knowledge-visualization/index.md

```diff
@@ -0,0 +1,59 @@
+
+### Knowledge graphs and visualizations
+Knowledge visualization involves using visuals like graphs, sketches, images, and diagrams to present information. Often there is confusion on whether knowledge graphs and knowledge visualization are the same things expressed differently. That view is true to some extent, but it is an oversimplification.
+
+Knowledge graphs are not just another means of knowledge visualization but rather an elaboration of knowledge visualization. Knowledge visualization is different because it gives insight into data after its complete analysis. It presents the final phase of data processing. But knowledge graphs show the entire process from linking the nodes, establishing their relationship, sharing data, analysis, and visual presentation. Essentially, knowledge visualization is a summarized portion of knowledge graphs.
+
+Knowledge visualization needs knowledge graphs to be at the core of its strategy. Knowledge graphs can integrate heterogeneous data sources extensively due to their semantic networks. Knowledge visualization lack such capability. Knowledge graphs can use semantic enrichment knowledge to improve data quality and standardize it for a given entity. Such modified changes in data influence knowledge visualization because they change the appearance of data after its analysis.
+
+### Knowledge graphs as a solution to machine learning problems
+The [Cambridge Analytica scandal](https://www.nytimes.com/2018/04/04/us/politics/cambridge-analytica-scandal-fallout.html) illustrates why modern machine learning methods need to become more transparent in their decisions. To put it in context, that 2016 scandal tarnished the reputation of the American democratic process and over 200 others globally after the company confessed to influencing the voter's choices on the ballot.
+
+Solutions like [DARPA's explainable AI (XAI)](https://www.darpa.mil/program/explainable-artificial-intelligence) program have been launched to create more trustable systems. But the explainable machine learning methods still fail to account for the users' context or background knowledge. That means it can be problematic to use these machine learning methods to control life-critical problems, such as healthcare. Knowledge graphs help eliminate these shortcomings because they provide a valuable form of domain-specific, machine-readable knowledge through their semantic enrichment. Their connected datasets provide background knowledge for artificial intelligence systems to better explain their decisions to the users.
```

Comment:
```suggestion
Solutions like [DARPA's explainable AI (XAI)](https://www.darpa.mil/program/explainable-artificial-intelligence) program have been launched to create more trustable systems. However, the explainable machine learning methods still fail to account for the users' context or background knowledge. This means it can be problematic to use these machine learning methods to control life-critical problems such as healthcare. Knowledge graphs help eliminate these shortcomings because they provide a valuable form of domain-specific, machine-readable knowledge through their semantic enrichment. Their connected datasets provide background knowledge for artificial intelligence systems to better explain their decisions to the users.